en/config_mc/README.md

```diff
@@ -1,10 +1,152 @@
+
+First [select a multicopter airframe](../config/airframe.md) (options are listed in [Airframe Reference > Copter](../airframes/airframe_reference.md#copter)).
+You should select the frame that matches your vehicle brand and model if one exists, and otherwise select the "Generic" frame type that most closely matches your geometry in terms of number of motors and their relative positions.
+For example, for a [Quadrotor X](../airframes/airframe_reference.md#quadrotor-x) frame you would look for the name of your frame in the list, and if it was not present select the [Generic Quadrotor X](../airframes/airframe_reference.md#copter_quadrotor_x_generic_quadcopter) frame.
+
+
+:::note
+Any selected multicopter frame can be modified in the next step (actuator configuration) to add/remove motors and otherwise changed the geometry.
+Selecting a frame that matches your vehicle reduces the configuration work required.
+
+:::details How does this work (details)
+Selecting an airframe applies a [frame configuration file](../dev_airframes/adding_a_new_frame.md#adding-a-frame-configuration) that contains a predefined set of [parameters](../advanced_config/parameters.md), such as [CA_AIRFRAME=0](../advanced_config/parameter_reference.md#CA_AIRFRAME) for the vehicle type and [CA_ROTOR_COUNT](../advanced_config/parameter_reference.md#CA_ROTOR_COUNT) for the number of rotors.
+
+Frames named with model/brand will define the vehicle type, number of rotors, actual rotor positions, and motor directions.
+They might also contain tuning and calibration parameters that are good enough to fly or at least auto-tune the vehicle "out of the box".
+They don't include the output mappings, unless the frame is pre-wired and non modifiable (so you would have to set this yourself in the next step).
+  
+The frames named with "Generic" generally just define the vehicle type, number of rotors, and "placeholder" rotor positions; almost everything needs to be configured.
+:::
+
+The next step is to define your vehicle [geometry](../config/actuators.md#motor-geometry-multicopter) (the number of motors and their relative positions) and [assign those motors](../config/actuators.md#actuator-outputs) to the physical outputs that they are wired to on your flight controller (both of these are covered in [Actuator Configuration and Testing](../config/actuators.md)).
+
+If using PWM ESCs and OneShot ESCs (but not DShot and DroneCAN/Cyphal ESC) you should then perform [ESC Calibration](../advanced_config/esc_calibration.md) before proceeding to [Motor Configuration](../config/actuators.md#motor-configuration).
+This ensures that all ESC provide exactly the same output for a given input (ideally we'd calibrate ESCs first, but you can't calibrate your ESCs until outputs are mapped).
+
+The final step is [Motor Configuration](../config/actuators.md#motor-configuration), which sets the directions of motors, and their disarmed, armed and high limits.
```

Comment:
```suggestion
The final step is [Motor Configuration](../config/actuators.md#motor-configuration), which for PWM ESCs sets the disarmed, low and high PWM limits.
```

Reply:
The direction cannot be configured for normal PWM ESC.

Reply:
Yes, but my understanding is that you still need to set the limits even for non-PWM ESC. Please correct me if I'm wrong. Assuming I am not, how about:

```suggestion
The final step is [Motor Configuration](../config/actuators.md#motor-configuration), which sets the motor input values for  disarmed, low and high speed.
For CAN and D-Shot it can also set motor direction.
```

Reply:
I'm not sure. Also the GUI implies that setting the direction there is not possible. 
![image](https://github.com/PX4/PX4-user_guide/assets/26798987/262c2943-093e-4459-8390-65707c0aeb96)
![image](https://github.com/PX4/PX4-user_guide/assets/26798987/1f5de8aa-be55-4427-8a0b-3dc5cbfd6b5b)

@bkueng can you enlighten us? In https://docs.px4.io/main/en/config/actuators.html#reversing-motors it says there is a "Set Spin Direction " button, where is that?

Reply:
> That seems very unintuitive to me

It's a bit of a compromise for an efficient UI layout. I see that point though. Maybe adding 'Permanently' is enough?

> It feels wrong because this will change the directions of all ESCs presumably

No, you can set them individually (there's a dropdown list when clicking the button).

> you can set the direction of CAN or DSHOT ESC through the UI but not PWM.

Only DShot, CAN would still need to be implemented.

> You should ESC calibrate for PWM and One SHot but not Dshot and Can

Yes

> For all ESC you still need to set disarmed, armed, and high values.

It depends. For dshot you don't need to set them, but there's DSHOT_MIN if needed.

Reply:
Thanks very much @bkueng - I'll adjust the text appropriately.

>>It feels wrong because this will change the directions of all ESCs presumably
> No, you can set them individually (there's a dropdown list when clicking the button).

I don't think this is ever going to be intuitive, but it might be convenient. if it were me I'd have toggle button next to each affected motor - the UI would have to be special though in that the button would hold not have visible - you press it, that changes the state, but it isn't a radio button or similar that shows selected/not selected.

Reply:
Modifying THIS text to be correct. Will separately update actuator config. @sfuhrer FYI

```suggestion
The final step is [Motor Configuration](../config/actuators.md#motor-configuration):
- [Reverse any motors](../config/actuators.md#reversing-motors) that don't match the spin direction configured in the Geometry.
  For DShot ESC you can do this through the Acuator Testing UI.
- PWM, OneShot, and CAN ESC, set the motor input limits for disarmed, low and high speed (not needed for DShot ESC)
```